src/test/java/com/aws/greengrass/lifecyclemanager/LogManagerHelperTest.java

```diff
@@ -83,31 +91,20 @@ void setup() {
 
     @AfterEach
     void cleanup() {
-        LogConfig.getInstance().reset();
-        LogConfig.getInstance().closeContext();
+        LogConfig.getRootLogConfig().reset();
+        LogConfig.getRootLogConfig().closeContext();
         for (LogConfig logConfig : LogManager.getLogConfigurations().values()) {
             logConfig.reset();
             logConfig.closeContext();
         }
         LogManager.getTelemetryConfig().reset();
     }
 
-    @BeforeAll
-    static void setupLogger() {
-        LogManager.getRootLogConfiguration().setStore(LogStore.FILE);
-        LogManager.getTelemetryLogger(METRIC_LOGGER_PREFIX + "test");
-    }
-    @AfterAll
-    static void cleanupLogger() {
-        LogManager.getTelemetryConfig().setStore(LogStore.CONSOLE);
-        LogManager.getRootLogConfiguration().setStore(LogStore.CONSOLE);
-    }
-
     @Test
     void GIVEN_mock_service_WHEN_getComponentLogger_THEN_logs_to_correct_log_file() throws IOException {
         when(mockGreengrassService.getServiceName()).thenReturn("MockService");
 
-        LogConfig.getInstance().setStore(LogStore.FILE);
+        LogConfig.getRootLogConfig().setStore(LogStore.FILE);
```

Comment:
Reset the logger after the test otherwise we won't get logs from our tests.

Reply:
Added back @AfterAll for this.